std/http/file_server.ts

```diff
@@ -118,26 +118,22 @@ function fileLenToString(len: number): string {
 export async function serveFile(
   req: ServerRequest,
   filePath: string
-): Promise<Response> {
+): Promise<ServerResponse> {
   const [file, fileInfo] = await Promise.all([open(filePath), stat(filePath)]);
-  const headers = new Headers();
-  headers.set("content-length", fileInfo.size.toString());
-  const contentTypeValue = contentType(filePath);
-  if (contentTypeValue) {
-    headers.set("content-type", contentTypeValue);
-  }
-  return {
-    status: 200,
+
+  return new ServerResponse({
     body: file,
-    headers,
-  };
+    headers: {
+      "content-type": contentType(filePath) ?? fileInfo.size.toString(),
```

Comment:
Shouldn't `fileInfo.size` be the value of `content-length` header?

Reply:
I think it should.

Reply:
It is provided as `content-type` header.